.github/workflows/python-package.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
It seems like [Python 3.7 fails](https://github.com/ACM-Research/online-portfolio-selection/pull/13/checks?check_run_id=2222799067) due to some keywords not being defined in that version. You can go ahead and remove it from this `python-version` array